packages/react/lib/Builder/index.test.tsx.snap

```diff
@@ -150,6 +150,22 @@ exports[`<Builder /> should allow to add content from catalogue: Catalogue opene
                   Foldable
                 </span>
               </a>
+              <a
```

Comment:
how you did to change the snap without touching test file ?

Reply:
there's a new component so the catalogue content changes

Reply:
thats so evident... sorry Im dumb